src/mapping/PetRadialBasisFctMapping.hpp

```diff
@@ -613,7 +641,7 @@ void PetRadialBasisFctMapping<RADIAL_BASIS_FUNCTION_T>::mapConsistent(DataID inp
 
   // For every data dimension, perform mapping
   for (int dim = 0; dim < valueDim; dim++) {
-    printMappingInfo(inputDataID, dim);
+    // TODO printMappingInfo(inputDataID, dim);
```

Comment:
Reminder

Reply:
This one is tricky. We don't have information about the data apart from the dimensionality anymore.

Is this print worth breaking the abstraction? Can we do this differently?

Reply:
> Is this print worth breaking the abstraction?

No, probably not.

We could print the data names before call `mapData` etc? How do we do this for other mappings? Why do we only run into problems for `PetRadialBasisFctMapping`?

Reply:
We could print the data name in the DataContext either along the sample or not.
Big question is where the mesh info should go.

Examples for mapping 2 samples of 2D data DataA from MeshA to MeshB:

## Separate sample logs without mesh info
```
Mapping DataA
Mapping sample t=1
Mapping from "MeshA" to "MeshB" for dimension 0 with polynomial set to consistent
Mapping from "MeshA" to "MeshB" for dimension 1 with polynomial set to consistent
Mapping sample t=2
Mapping from "MeshA" to "MeshB" for dimension 0 with polynomial set to consistent
Mapping from "MeshA" to "MeshB" for dimension 1 with polynomial set to consistent
```

## Sample logs without mesh info
```
Mapping DataA sample at t=1
Mapping from "MeshA" to "MeshB" for dimension 0 with polynomial set to consistent
Mapping from "MeshA" to "MeshB" for dimension 1 with polynomial set to consistent
Mapping DataA sample at t=2
Mapping from "MeshA" to "MeshB" for dimension 0 with polynomial set to consistent
Mapping from "MeshA" to "MeshB" for dimension 1 with polynomial set to consistent
```

## Separate sample logs with Mesh info
```
Mapping DataA from MeshA to MeshB
Mapping sample t=1
Mapping dimension 0 with polynomial set to consistent
Mapping dimension 1 with polynomial set to consistent
Mapping sample t=2
Mapping dimension 0 with polynomial set to consistent
Mapping dimension 1 with polynomial set to consistent
```

## Sample logs with Mesh info
```
Mapping DataA sample t=1 from MeshA to MeshB
Mapping dimension 0 with polynomial set to consistent
Mapping dimension 1 with polynomial set to consistent
Mapping DataA sample t=2 from MeshA to MeshB
Mapping dimension 0 with polynomial set to consistent
Mapping dimension 1 with polynomial set to consistent
```

Reply:
I tried to condense the output as much as possible.
```
17:26:27.519149|Ateles|0|impl::DataContext|l104|mapData|Mapping "Pressure" t=1 from "FASTEST_Mesh" to "Ateles_Mesh"
17:26:27.519185|Ateles|2|mapping::NearestNeighborMapping|l60|mapConsistent|Map consistent
```

Or for a PET RBF mapping:
```
17:26:28.296830|SolverOne|2|mapping::PetRadialBasisFctMapping|l900|printMappingInfo|Mapping consistent for dimension 0 with polynomial set to separate
```

`t=1` is the stample for the timestamp 1. This way one can see the individual mappings of all samples.

The kind of mapping is part of the log information, not the log message though.
We could add it to the DataContext log or the Mapping log:

```
Mapping "Pressure" t=1 from "FASTEST_Mesh" to "Ateles_Mesh" using a NearestNeighbour mapping
Map consistent
```

or

```
Mapping "Pressure" t=1 from "FASTEST_Mesh" to "Ateles_Mesh"
Map consistent using NearestNeighbour
```

Reply:
I implemented version 2 to keep this a bit more balanced.